test/base/BaseSpec.scala

```diff
@@ -19,9 +19,16 @@ package base
 import org.scalatest.concurrent.ScalaFutures
 import org.scalatest.{MustMatchers, WordSpec}
 import play.api.libs.json._
+import play.twirl.api.Html
+import org.jsoup._
+import org.jsoup.nodes.{Document, Element}
+import scala.collection.JavaConverters._
 
 trait BaseSpec extends WordSpec with MustMatchers with ScalaFutures {
 
+  def asDocument(html: Html): Document = Jsoup.parse(html.toString)
```

Comment:
I would have put these two methods in ViewSpec as their use is mainly confined to tests of views.

Reply:
will move